ext/mbstring/php_mbregex.c

```diff
@@ -73,6 +74,7 @@ static int _php_mb_regex_globals_ctor(zend_mb_regex_globals *pglobals)
 {
 	pglobals->default_mbctype = ONIG_ENCODING_UTF8;
 	pglobals->current_mbctype = ONIG_ENCODING_UTF8;
+	pglobals->current_mbctype_mbfl_encoding = mbfl_name2encoding("UTF-8");
```

Comment:
It is better to use `&mbfl_encoding_utf8` after including the appropriate header.